### PR TITLE
CORE-1687 Replace Subscription Buy URL with config value

### PR DESCRIPTION
**Pull request by psarando**
This PR will replace the subscription `Buy` button's URL with a config value instead of using a constant.

### CODE REVIEW

**Review by johnworth — APPROVED** [review]
LGTM :shipit: 

Thanks for working on this.

File: stories/configMock.js [hunk 1]
```diff
@@ -42,10 +42,8 @@ export default {
         defaultCasUrl: "https://olson.cyverse.org/cas",
         defaultCasValidate: "validate",
     },
-    metadata: {
-        communityAttr: "cyverse-community",
-        instantLaunchLocationAttr: "ui_location",
-        instantLaunchDashboard: "dashboard",
-        instantLaunchNavDrawer: "nav_drawer",
```
Comment (ianmcorvidae) [inline stories/configMock.js:49]: Hopefully removing these from the configMock is fine? doesn't seem like they moved somewhere else

**Review by ianmcorvidae — APPROVED** [review]
Looks basically reasonable to me! One small question, as I usually end up doing, heh.

File: stories/configMock.js (hunk 1 above)
Comment (psarando) [reply stories/configMock.js:49]: Yep, I happened to notice that the actual configs were removed in a previous PR, so I thought I'd clean these up while I was making changes here.

**Comment by psarando** [comment]
Thanks for the reviews! 👍

**Pull request merged**